fla/ops/log_linear_attn/chunk.py

```
@@ -0,0 +1,1208 @@
+        for num_warps in [4]
+        for num_stages in [2, 3, 4]
+    ],
+    key=["H", "K", "V"],
+)
+@triton.jit(do_not_specialize=["T"])
+def chunkwise_fwd_kernel(
+    q,
+    k,
+    v,
+    g,
+    l,
```

Ruff (0.11.9)

Ambiguous variable name: `l`

(E741)